conda/cli/conda_argparse.py

```diff
@@ -31,18 +30,16 @@
     DepsModifier,
     UpdateModifier,
 )
-from ..base.context import context
+from ..base.context import context, sys_rc_path, user_rc_path
+from ..common.compat import on_win
 from ..common.constants import NULL
 from ..deprecations import deprecated
 from .find_commands import find_commands, find_executable
 
 log = getLogger(__name__)
 
-# duplicated code in the interest of import efficiency
-on_win = bool(sys.platform == "win32")
-user_rc_path = abspath(expanduser("~/.condarc"))
 escaped_user_rc_path = user_rc_path.replace("%", "%%")
-escaped_sys_rc_path = abspath(join(sys.prefix, ".condarc")).replace("%", "%%")
```

Comment:
We already import `context`, importing these other things wont be an additional hit